### DISCUSSION TITLE
Uhhh... Help?

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#502] [Q&A] [answered]
This feels like a creepypasta, but there's been an error where the main menu background so I checked assets and found this
![image](https://github.com/user-attachments/assets/3977d822-71af-4a03-bfed-5bc8ed06c507)
also whenever it pops up I get an error saying "libpng error: bad adaptive filter value"

UPDATE: it also spread to Spooky BG
![image](https://github.com/user-attachments/assets/4c0b426c-5b8b-4d4c-b4d5-e3fd9ab5a349)

**Comment by Moro-Maniac** [comment, marked as answer]
mb i ate them :3

okay but seriously, just try redownloading the game. it's likely something corrupted during the download

**Reply by THEkatinamicrowave** [reply]
thanks yeah I redownloaded the assets folder

**Reply by THEkatinamicrowave** [reply]
oh yeah now I remember
I download all my FNF stuff to an external drive
this external drive is really finnicky and sometimes just disconnects for no reason
I think it disconnected mid-download and then I continued it after reconnecting
